src/main/java/io/fabric8/maven/docker/assembly/DockerAssemblyManager.java

```diff
@@ -176,7 +176,7 @@ public TarArchiver customize(TarArchiver archiver) throws IOException {
                         // Interpolated Dockerfile is already added as it was created into the output directory when
                         // using dir dir mode
                         excludeDockerfile(fileSet, dockerFile);
-
+                        fileSet.setUsingDefaultExcludes(buildConfig.defaultExclusion());
```

Comment:
We might consider to set this always to "false", as IMO this is the sensible behaviour (and usually it also doesn't harm except that it might increase the performance slightly).